nano/node/common.cpp

```diff
@@ -696,6 +696,21 @@ void nano::keepalive::serialize (nano::stream & stream_a) const
 	}
 }
 
+std::string nano::keepalive::to_string ()
+{
+	std::stringstream stream;
+
+	stream << "\n" + header.to_string () + "\n";
+
+	for (auto peer (peers.begin ()), j (peers.end ()); peer != j; ++peer)
```

Comment:
Use:
```
 auto peer = peers.begin ()
 ```

Reply:
done. Should I make this change in other places in the same file where they use `auto peer (peers.begin ())` as opposed to `auto peer = peers.begin ()`? Like, in `nano::keepalive::serialize` around line 690

Reply:
In general yes, but for now since this is your first PR, let's just focus on getting it through the review process and merged.